src/gc/env/gcenv.base.h

```diff
@@ -551,6 +551,7 @@ void LogSpewAlways(const char *fmt, ...);
 void StompWriteBarrierEphemeral(bool isRuntimeSuspended);
 void StompWriteBarrierResize(bool isRuntimeSuspended, bool bReqUpperBoundsCheck);
 bool IsSuspendEEThread();
```

Comment:
it's already removed in 1.0.0.0. so I left this one alone... do we have some sort of reverse integration between these git branches?